src/api.v2/controllers/experimentController.js

```diff
@@ -153,11 +154,16 @@ const cloneExperiment = async (req, res) => {
   // we want to start without any old tracks so clear them out
   await new MetadataTrack().delete({ experiment_id: toExperimentId });
 
-  // Get samples order so we preserve the relative order of the original samples in the cloned ones
-  const { samplesOrder } = await new Experiment().findById(fromExperimentId).first();
+  let samplesToCloneIds = samplesSubsetIds;
```

Comment:
Can you rename this during import at line 148 above? (i.e. `body: { samplesSubsetIds : samplesToCloneIds = null }`), example: https://stackoverflow.com/a/56700589/1940886